test/parallel/test-util-inherits.js

```diff
@@ -75,6 +75,12 @@ assert.strictEqual(e.e(), 'e');
 assert.strictEqual(e.constructor, E);
 
 // should throw with invalid arguments
-assert.throws(function() { inherits(A, {}); }, TypeError);
-assert.throws(function() { inherits(A, null); }, TypeError);
-assert.throws(function() { inherits(null, A); }, TypeError);
+assert.throws(function() {
+  inherits(A, {});
+}, /^TypeError: The super constructor to "inherits" must have a prototype$/);
+assert.throws(function() {
+  inherits(A, null);
+}, /^TypeError: The super constructor to "inherits" must not be null or undefined$/); // eslint-disable-line max-len
```

Comment:
Specifically, I know this inlines the eslint rule, but this can be reworked to avoid that entirely...

```js
const errCheck =
  new RegExp('^TypeError: The super constructor to "inherits" must not be ' +
                      'null or undefined$');
assert.throws(() => inherits(A, null), errCheck);
```